entries/c/cloud.oracle.com.json

```
@@ -1,6 +1,9 @@
 {
   "Oracle Cloud Infrastructure": {
-    "domain": "oracle.com",
+    "domain": "cloud.oracle.com",
+    "additional-domains": [
+      "oraclecloud.com"
+    ],
```

Since the entry can't redirect to a login page I think redirecting to this page would be the best fit:
```suggestion
    ],
    "url": "https://www.oracle.com/cloud/",
```